src/script/main/app.js

```diff
@@ -388,7 +388,7 @@ class App {
       telemetry.time_step(AppInitTimingsStep.VALIDATED_CLIENT);
       telemetry.add_statistic(AppInitStatisticsValue.CLIENT_TYPE, clientEntity.type);
 
-      await cryptographyRepository.loadCryptobox(this.service.storage.db);
+      await cryptographyRepository.initCryptobox(true);
```

Comment:
default value is `true`
```suggestion
      await cryptographyRepository.initCryptobox();
```

Reply:
I don't want to rely on default values. I am actually considering removing the default.